### ISSUE TITLE
[suggestion] placeholder

### ISSUE THREAD

**Issue by diznover** [modrinth-smp/fabric-rep-system#2]
Can you add a placeholder for the reputation of the players? This would allow you to see the player's reputation in the tab list or in the chat.

**Comment by Gaming32** [comment]
Placeholder? Are you referring to PlaceholderAPI? I thought that was a Spigot/Paper thing.

**Comment by ghost** [comment]
the kind of thing where you would type %rep% and it would show the rep as a number

**Comment by Gaming32** [comment]
Placeholders can be done currently if you're willing to write some commands. Specifically `execute store`, `rep view`, and `tellraw`.

**Comment by diznover** [comment]
Yes it is also there for fabric. Here is a link to the placeholderApi for fabric [TextPlaceholderAPI](https://github.com/Patbox/TextPlaceholderAPI), and mods in which placeholders can be displayed [StyledPlayerList](https://github.com/Patbox/StyledPlayerList) [StyledChat](https://github.com/Patbox/StyledChat)
